test/system/index.js

```diff
@@ -1,15 +1,18 @@
-const trader = require('./trader');
+const fs = require('fs');
+const Promish = require('promish');
+const Trader = require('./trader');
 const Commander = require('./commander');
-const initTraders = require('./initTraders');
 const TestSingleTrader = require('./scenarios/testSingleTrade');
 const TestTwoTrader = require('./scenarios/testTwoTrader');
 const TestBalances = require('./scenarios/testBalances');
-const fs = require('fs');
+const systemCleanup = require('./systemCleanUp');
+const result = require('./systemTestResult');
+// const initTraders = require('./initTraders');
```

Comment:
Do we no longer need to initTraders? If so, remove this line and the initTraders.js file